src/wp-includes/post.php

```diff
@@ -588,8 +588,12 @@ function create_initial_post_types() {
 			),
 			'map_meta_cap'                   => true,
 			'query_var'                      => false,
-			'show_in_rest'                   => false,
 			'rewrite'                        => false,
+			'show_in_rest'                   => true,
+			'rest_base'                      => 'font-families',
+			'rest_controller_class'          => 'WP_REST_Font_Families_Controller',
+			// Disable autosave endpoints for font families.
+			'autosave_rest_controller_class' => 'stdClass',
```

Comment:
**Aside:**

We should really find a better way to avoid hacks like this. I don't think this is a clean way at all, and devs will just copy this sooner or later.

If workarounds like this are added/needed in GB it's a sign that we should improve core, not just bring the workaround into core.

Maybe it could be handled via post type supports, i.e. `add_post_type_support( 'autosave' )` / `remove_post_type_support( 'autosave' )`. I suppose https://core.trac.wordpress.org/ticket/41172 could be used to track this, maybe I can take a stab at it.